codeGenerator/src/FRCrobotCodeGen302/MainForm.cs

```diff
@@ -39,11 +39,26 @@ public MainForm()
             valueTextBox.Location = valueNumericUpDown.Location;
 
             this.Text += " Version " + ProductVersion;
+
+            //try to load cached configuration.xml
+            addProgress("Trying to load cached configuration.xml");
+            if (File.Exists(Path.GetTempPath() + "DragonsCodeGeneratorCache.txt"))
+            {
+                string cachedPath = File.ReadAllText(Path.GetTempPath() + "DragonsCodeGeneratorCache.txt");
+                configurationFilePathNameTextBox.Text = cachedPath;
+                loadConfiguration(cachedPath);
+                addProgress("Loaded cached configuration.xml");
+                robotConfigurationFileComboBox_TextChanged(null, null);
+            }
+            else
+            {
+                addProgress("Failed to load cached configuration.xml");
```

Comment:
This message should say that the cache file does not exist and therefore the robot configuration will not be loaded